src/IssueViz.Security/Hotspots/IHotspotMatcher.cs

```diff
@@ -0,0 +1,30 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarLint.VisualStudio.IssueVisualization.Models;
+using SonarQube.Client.Models;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.Hotspots
+{
+    internal interface IHotspotMatcher
```

Comment:
Is there a no-op MEF-exported implementation of this interface? If not, anything that depends on it will fail at runtime. Since this now includes the `IssueConsumerFactory` it presumably means that analysis won't work.

This would probably be pretty confusing/frustrating for anyone else syncing the branch while working on the sprint.